meshio/helpers.py

```diff
@@ -205,7 +205,8 @@ def write(filename, mesh, file_format=None, **kwargs):
 
     # check cells for sanity
     for key, value in mesh.cells.items():
-        assert value.shape[1] == num_nodes_per_cell[key]
+        if "polygon" not in key:
```

Comment:
This should be `key != "polygon"` to make sure the rest is tested.

Reply:
Yes, but I do not want the rest to be tested as, for now, the key is "polygonN" with N the number of vertices/edges

Reply:
If the key `polygonN`, then this conditional is always true. I think I misunderstand something here.

Reply:
Yes, that's the point all polygon have the same VTK_TYPE

Reply:
Ah, okay. For clarity, this should be replaced by `key[:7] == "polygon"`.

Reply:
Why not use the pythonic way: ``key.startswith("polygon")``?

Reply:
Because my python is quite new, what's your opinion @nschloe?